chirp/drivers/kenwood_live.py

```diff
@@ -84,7 +84,10 @@ def command(ser, cmd, *args):
 
     result = ""
     while not result.endswith(LAST_DELIMITER[0]):
-        result += ser.read(COMMAND_RESP_BUFSIZE).decode()
+        try:
+            result += ser.read(COMMAND_RESP_BUFSIZE).decode()
+        except:
```

Comment:
I recommend never using "bare except". Something is definitely wrong here.

Reply:
Yes, this should be `except Exception` at the least. That said, the bytes/str sort of errors this is obviously meant to cover up is exactly what is tricky about these conversions and why it's not as simple as just 2to3.

IMHO, this sort of thing can't be in anything we merge here.